lib/components/dropdown.vue

```diff
@@ -30,7 +31,7 @@
         <div :class="['dropdown-menu',{'dropdown-menu-right': right}]"
              ref="menu"
              role="menu"
-             :aria-labelledby="split ? null : _id"
+             :aria-labelledby="id ? (id + (split ? '__BV_toggle_' : '__BV_button_')) : null)"
```

Comment:
I am think this might be where it is, although I have looked at it 20 times and don't see any typos.

Maybe if I move this to a computed property